### PR TITLE
Misc log improvements

### PR DESCRIPTION
**Pull request by maaktweluit**
During my last task: #3189 these fixes did not make it to the PR in time.

Split them in this separate PR:
- Add uniform way to print node_name + node_id
- Add log messages in `tasksession._react_to_wants_to_compute`

### CODE REVIEW

File: tests/golem/test_client.py [hunk 1]
```diff
@@ -820,9 +822,11 @@ def test_run(self):
         client.clean_old_tasks.assert_called_once()
 
 
+@patch('golem.client.node_info_str')
 @patch('signal.signal')
 @patch('golem.network.p2p.node.Node.collect_network_info')
 class TestClientRPCMethods(TestWithDatabase, LogTestCase):
+    # pylint: disable=too-many-public-methods
```
Comment (shadeofblue) [inline tests/golem/test_client.py:829]: it would probably be good to refactor these tests instead...

**Review by shadeofblue — APPROVED** [review]

**Review by Wiezzel — APPROVED** [review]
I prefer using f-strings over `%` or `{}` formatting, but it's OK.

File: tests/golem/test_client.py (hunk 1 above)
Comment (maaktweluit) [reply tests/golem/test_client.py:829]: Will have a look if i can split it up, if it takes too much time i'll make an issue to fix this later

**Comment by maaktweluit** [comment]
@Wiezzel when using logs `%` is the best way, because it will format only when the level is active.

For this one i'll change to f'' strings, agree this will look neater.

File: tests/golem/test_client.py (hunk 1 above)
Comment (maaktweluit) [reply tests/golem/test_client.py:829]: Looked at it and do not see an easy split, most of them are testing RPC methods..

shall i make a separate issue to look into this or leave as is, there are many more tests that need cleanup like this.

**Pull request merged**